lib/child_process.js

```diff
@@ -566,8 +566,7 @@ function checkExecSyncError(ret, args, cmd) {
     err = new Error(msg);
   }
   if (err) {
-    err.status = ret.status < 0 ? errname(ret.status) : ret.status;
```

Comment:
Is this logic being lost by moving to `Object.assign()`?

Reply:
Yes but I don't think this logic should be there. It was added in https://github.com/nodejs/node/pull/13601.
The documentation says: "The Error object will contain the entire result from child_process.spawnSync()". If we want to translate the status, it should be done in `spawnSync` and `execSync` will inherit it.